app/styles/components/_buttons.scss

```diff
@@ -1,17 +1,17 @@
 .btn {
   &.on-white {
-    background-color: $blue_light;
-    color: $navy;
+    background-color: #08987f;
+    color: $white;
```

Comment:
I have swapped the hover style with active state style of Add button in the third commit.
- Active version is lighter version as default state
- Hover version is darker version
please check review the commit and suggest other invaluable changes

active state - 
![acive](https://cloud.githubusercontent.com/assets/16955978/23690313/8dd94940-03e6-11e7-8cea-0e252c3db452.png)

hover state -
![hover](https://cloud.githubusercontent.com/assets/16955978/23690324/adbb50f0-03e6-11e7-9bd2-d54cc1ad0217.png)